rest-api/dao/participant_summary_dao.py

```diff
@@ -379,6 +380,34 @@ def to_client_json(self, model):
 
     return result
 
+  def to_client_csv(self, model):
+    from flask import Response
+    import StringIO
+    # from unicode_csv import UnicodeWriter
+
+
+    def iter_csv(data):
+      line = StringIO.StringIO()
+      writer = csv.writer(line)
+      for csv_line in data:
+        writer.writerow(csv_line)
+        # line.seek(0)
+        # yield line.read()
+        # line.truncate(0)
+
+    response = Response(iter_csv(model), mimetype='text/csv')
```

Comment:
A Response would need to be returned from participant_summary_api.get()... we're not making one Response per result, if that makes sense.

Similarly, there should only be one CSV writer, which would get passed in to to_client_csv for each participant summary.